### ISSUE TITLE
unable to connect to Travis using Github login with 2FA

### ISSUE THREAD

**Issue by mobilemind** [dimroc/iOS.ProjectMonitor#1]
I am unable to login with valid credentials. I get an error

```
Unable to talk to Github    
Request failed: unauthorized (401).
```
1. Does iOS.ProjectMonitor support login to Gituhub with 2FA? I use it.
2. Does iOS.ProjectMonitor properly encode special characters? I have a complex password that may include characters like `!@#$%^&*+|<>/\\'"`. Some apps have trouble with these characters.

**Comment by dimroc** [comment]
Hi Tom, thanks for checking out the app. Answers below:
1. Doubtful. I have not tried two factor authentication.
2. Yes it does. I just tried it with `!@#$%^&*+|<>/\\'"`.

The 401 returned is [mentioned here in the Github API documentation for 2FA](https://developer.github.com/v3/auth/#working-with-two-factor-authentication).

Bandwidth is limited at the moment, but I'll try to take a look. Contributions are welcome!

**Comment by mobilemind** [comment]
Consider #1 confirmed; no 2FA support, (yet?)

**Comment by nertzy** [comment]
I have the same issue. Can't connect via 2FA. Also I tried generating an API token and using that, but alas it also didn't work. Hi @dimroc!

**Comment by lhunath** [comment]
Same issue. There's no chance I'm turning off 2FA; and this is getting increasingly encouraged. Please look into fixing this.

**Comment by dimroc** [comment]
Thanks for all the interests guys. 2FA support has just been released in 1.0.5 and should be available in the app store as of today.

Next time I'm in Austin @nertzy, I'm ordering on your tab!

**Comment by nertzy** [comment]
:+1: :beers:
